doc/modules/language-guide/pages/basic-concepts.adoc

```diff
@@ -4,7 +4,9 @@
 
 {proglang} is designed for distributed programming with actors.
 
-Before you begin writing programs for distributed processing using actors, you should be familiar with a few of the basic building blocks of any programming language and with {proglang} in particular. 
+ When programming on the Internet Computer in Motoko, each **actor** represents an **{IC} canister smart contract** with a Candid interface, whether written in Motoko, Rust, Wasm or some other language that compiles to Wasm. Within Motoko, we use the term *actor* to refer to any canister, authored in any language that deploys to the {IC}. The role of Motoko is to make these actors easy to author, and easy to use programmatically, once deployed.
```

Comment:
Maybe "once deployed, easy to use programmatically" is easier to read? It's probably just a familiarity thing so feel free to ignore

Reply:
Should the `Motoko`s in this line be `{proglang}`? There are 4 occurrences in this line.

Reply:
```suggestion
 When programming on the Internet Computer in {proglang}, each **actor** represents an **{IC} canister smart contract** with a Candid interface, whether written in {proglang}, Rust, Wasm or some other language that compiles to Wasm. Within {proglang}, we use the term *actor* to refer to any canister, authored in any language that deploys to the {IC}. The role of {proglang} is to make these actors easy to author, and easy to use programmatically, once deployed.
```
God forbid  we ever change the name again... Motoko smart blockchain instruction vernacular anyone?